src/UnityExtension/Assets/Editor/GitHub.Unity/UI/Window.cs

```diff
@@ -326,7 +326,8 @@ private void DoAccountDropdown()
 
         private void SignIn(object obj)
         {
-            AuthenticationWindow.Open();
+            var popupWindow = (PopupWindow) PopupWindow.Open(PopupWindow.PopupView.AuthenticationView);
+            popupWindow.Initialize(EntryPoint.ApplicationManager);
```

Comment:
I was looking at it a bit more, I was returning `IView` out of copy-pasting older code, since there is now only one type of popup being `PopupWindow`, I figured the easiest thing to do is return `PopupWindow` from the `Open` method.